plugins/org.jkiss.dbeaver.ui.editors.connection/src/org/jkiss/dbeaver/ui/dialogs/connection/AuthModelSelector.java

```diff
@@ -164,6 +167,25 @@ public void loadSettings(
         changeAuthModel();
     }
 
+    @Override
+    public void handleDataSourceEvent(DBPEvent event) {
+        if (event.getAction() == DBPEvent.Action.OBJECT_UPDATE) {
+            UIUtils.asyncExec(() -> {
+                if (event.getData() instanceof DBPConnectionConfiguration newConfig) {
```

Comment:
Can be merged to just one `if`

```suggestion
        if (event.getAction() == DBPEvent.Action.OBJECT_UPDATE && event.getData() instanceof DBPConnectionConfiguration newConfig) {
            UIUtils.asyncExec(() -> {
```